app/controllers/admin_general_controller.rb

```diff
@@ -114,8 +114,7 @@ def timeline
       end
       # get all the models in the slice, eagerly loading the associations we use in the view
       public_body_versions = PublicBody.versioned_class.
-        includes(:public_body => :translations).
-          find(public_body_version_ids.keys)
+        find(public_body_version_ids.keys)
```

Comment:
Don't know what this is doing.

If we need it, then:
- tests
- commit message explaining why we need the change.